config/enterprise_versions.yml

```diff
@@ -85,6 +85,9 @@ components:
   packetcapture-api:
     image: tigera/packetcapture-api
     version: master
+  tigera-prometheus-service:
+    image: tigera/prometheus-service
```

Comment:
yeah think really screwed up with the name for this, since semaphore is already deploying the image as prometheus-service and the repo is of the same name,  but that's basically why, since it might become a full fledged service / api for prometheus with RBAC and more endpoints than a proxy

we should find a common name and change it everywhere 

not sure if you're ok with `tigera-prometheus-api` or agree on a different name @tmjd

Reply:
This is fine since the image/repo/ci has already been created. Don't worry about changing it.